src/components/Rout.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {
+  BrowserRouter as Router, Switch, Route, Link,
+} from 'react-router-dom';
+import './Rout.css';
+import Calculator from './Calculator';
+
+export default function Rout() {
+  return (
+    <Router>
+      <nav>
+        <ul>
+          <li>
+            <Link to="/">Home</Link>
+          </li>
+          <li>
+            <Link to="/Calculator">Calculator</Link>
+          </li>
+          <li>
+            <Link to="/Quote">Quote</Link>
+          </li>
+        </ul>
+      </nav>
+      {/* Used switch component to display different content on UI */}
+      <Switch>
+        <Route exact path="/">
+          <Home />
+        </Route>
+        <Route path="/Calculator">
+          <CalculatorPage />
+        </Route>
+        <Route path="/Quote">
+          <Quote />
+        </Route>
+      </Switch>
+    </Router>
+  );
+}
+
+function Home() {
+  const text = 'Welcome to Math Magicians, your one-stop destination for all things math!\n\n'
+    + 'Whether you\'re a student, a teacher, or just someone who loves math, '
+    + 'we\'ve got everything you need to enhance your math skills and knowledge. '
+    + 'From basic arithmetic to advanced calculus, we\'ve got it all covered. '
+    + 'So why wait? Start exploring our site today and take your math skills to '
+    + 'the next level!';
+  return (
+    <div className="Home-content">
+      <h1>Math Magicians</h1>
+      <h2>Welcome to our page!</h2>
+      <p>{text}</p>
+    </div>
+  );
+}
+function CalculatorPage() {
+  return (
+    <div className="calculator-content">
+      <h1>Math Magicians</h1>
```

Comment:
- [ ] Please let's ensure that our design follows the layout guideline of the wireframe correctly. Please notice that in the wireframe the calculator page has "Let's do some math" on the left and the Calculator on the right taking up the rest of the available screen.
### Wireframe guideline
![image](https://user-images.githubusercontent.com/46839548/221892841-3f1a8649-5e63-4ab2-8db0-f4ade7450b4f.png)

### Your implementation
![image](https://user-images.githubusercontent.com/46839548/221893277-dc32a814-e296-4c0c-b314-ed6a194046d9.png)

- [ ] The way you have set up all 3 component pages (`<Home />`, `<CalculatorPage />` and `<Quote />`) in the same `Rout.js` file works quite alright but it is neither professional nor clean, and doesn't make your code easy to read. You can see the potential of this file being too long if you were to do something more complex, but the added disadvantage of difficulty while debugging. Please extract all 3 pages into their own separate individual files 👍